.github/workflows/ci-cassandra.yml

```diff
@@ -30,17 +32,17 @@ jobs:
       run: bash scripts/cassandra-integration-test.sh ${{ matrix.version.schema }}
```

Comment:
`${{ matrix.version.schema }}` is not used in the script as far as I can tell. Instead it does this explicitly:
```
docker run --network integration_test -e CQLSH_HOST=cassandra -e TEMPLATE=/cassandra-schema/v001.cql.tmpl jaeger-cassandra-schema-integration-test
docker run --network integration_test -e CQLSH_HOST=cassandra2 -e TEMPLATE=/cassandra-schema/v002.cql.tmpl jaeger-cassandra-schema-integration-test
```

Reply:
I noticed that v003.cql.tmpl was never being tested as part of that script. Hence I am passing those 3 templates as args and referring them as $1, $2, $3 in the script.